glide.yaml

```diff
@@ -1,19 +1,19 @@
 package: github.com/openshift/openshift-azure
 import:
 - package: github.com/openshift/api
-  version: release-3.10
+  version: release-3.11
```

Comment:
Did you find that it's necessary to update our types and client to 3.11 in one go? A N-1 client should be compatible with a N server so it may not be necessary to do this update in this PR.

Reply:
No. It was a different issue, I just was eliminating possible cases.